src/Components/Shared/src/ExpressionFormatting/ExpressionFormatter.cs

```diff
@@ -14,7 +14,10 @@ internal static class ExpressionFormatter
 {
     static ExpressionFormatter()
     {
-        HotReloadManager.Default.OnDeltaApplied += ClearCache;
+        if (HotReloadManager.Default.MetadataUpdateSupported)
+        {
+            HotReloadManager.Default.OnDeltaApplied += ClearCache;
+        }
```

Comment:
nit: is there a reason to check for support instead of just adding the event handler?